COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=EVM server daemon
-After=syslog.target memcached.service
```

Comment:
Sorry if already discussed... I don't see why syslog was removed

Reply:
It wasn't discussed, but a good question.  There isn't a syslog.target on rhel/centos 7:
```
[root@manageiq ~]# systemctl status syslog.target
Unit syslog.target could not be found.
```

And we don't use syslog anyway (we use stdout / journald)

Reply:
ah, thanks... so was it quietly failing prior to this?

Reply:
Not failing exactly, just silently not doing anything

Reply:
Centos 8 too

Reply:
I can rhel8 to that